### DISCUSSION TITLE
Welcome to netscan Discussions!

### DISCUSSION THREAD

**Discussion by searge** [Searge/netscan#168] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dimension-hub** [comment]
Welcome men!

**Reply by searge** [reply]
![](https://media.tenor.com/images/244655c6c4106f3894b62bb1f5f27d2f/tenor.gif)

**Reply by dimension-hub** [reply]
I have to go to bed already😴
